.github/container/check-gpus.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+YELLOW='\033[0;33m'
+NOCOLOR='\033[0m'
+
+
+# Check that all devices are identical
+if [[ -z "$CUDA_VISIBLE_DEVICES" ]]; then
+    # CUDA_VISIBLE_DEVICES is not set. Check what GPUs are available
+    diff_gpu_count=$(nvidia-smi --query-gpu=gpu_name --format=csv,noheader | sort | uniq | wc -l)
+    if [[ $diff_gpu_count -ge 2 ]]; then
+            echo -e "${YELLOW}
+
+WARNING: There are $diff_gpu_count different GPUs in the system:
+
+$(nvidia-smi --query-gpu=gpu_name --format=csv,noheader | sort | uniq)
+
+Using all of them with JAX may
+cause the JAX failure. To avoid that you can either specify CUDA_VISIBLE_DEVICES env variable:
```

Comment:
```suggestion
Using all of them at once with JAX may cause JAX to fail. 
To avoid that you can either specify CUDA_VISIBLE_DEVICES env variable:
```